revscoring/utilities/union_merge_observations.py

```diff
@@ -40,36 +41,21 @@ def main(argv = None):
     else:
         out_file = open(args['--output'], "w")
 
-    transformation = DataUnion(id_column = args['--id-column'])
-    transformation.union(args['<input>'], out_file)
+    observations = []
+    for path in args['<input>']:
+        with open(path, "r") as in_file:
+            observations.extend(read_observations(in_file))
 
+    union_merge(observations, out_file, id_column = args['--id-column'])
```

Comment:
I'm not sure this will work outside of the `with ...` block because it should call `__exit__()` on the files as soon as the block is exited.